localdev/starters/example-starter/ocw-studio.yml

```diff
@@ -2,4 +2,5 @@ collections:
   - label: "Page"
```

Comment:
Should we convert this file to JSON if it's only going to be converted to JSON anyway? No strong feelings either way

Reply:
We're expecting the starter repos to define the config as yaml, so I think it makes sense to keep that pattern here